templates/uber-development.ini.erb

```diff
@@ -74,6 +74,14 @@ shirt_sales_enabled = <%= @shirt_sales_enabled %>
 show_affiliates = <%= @show_affiliates %>
 student_discount = <%= @student_discount %>
 
```

Comment:
in some future version, this stuff will be only config in CORE, and anythign else will be in its own .ini

for now, it does work to put things here as long as core is ok picking them up in it's Config data